### DISCUSSION TITLE
Disable "Back up only when charging" not possible

### DISCUSSION THREAD

**Discussion by Nurtyll** [seedvault-app/seedvault#659] [Q&A]
Hi, I bought a usb-c stick to back up my data to the stick. I set up Seedvault and now I cannot perform a backup because I cannot uncheck the "Backup only when charging" condition slider found inside the menu point "Backup scheduling". 
![screenshot](https://github.com/seedvault-app/seedvault/assets/75533058/e965bb85-befa-4344-918a-8ed9dd43bae4)

I understand that this is a safety feature to prevent the battery from running out during the backup. But my Fairphone 4 does not support wireless charging, so do I need to buy a split or can I somehow disable this feature?

I'm using the latest Calyxos for FP4.

Thanks for your help!

**Comment by chirayudesai** [comment]
What was the battery level when you took that screenshot?

**Comment by grote** [comment]
The backup conditions are only disabled, because they don't apply to USB backup locations. If you plug your flash drive in after 24h have passed, it should automatically do a backup. If you plug it in whenever, you should be able to manually trigger a backup.

**Reply by Nurtyll** [reply]
Thanks, I just saw that the app has already done a backup. I just was confused because somehow I couldn't manually trigger a backup. Anyway it should be working and thx again!
